pkgs/by-name/re/recon-ng/setup.py

```diff
@@ -0,0 +1,25 @@
+from setuptools import setup
+
+setup(
+    name='@pname@',
+    version='@version@',
+    install_requires=[
+        "pyyaml",
+        "dnspython",
+        "lxml",
+        "mechanize",
+        "requests",
+        "flask"
```

Comment:
```suggestion
        "flask",
```

Reply:
I can't believe nothing made this obvious, thanks